examples/userdemo/pgf_fonts.py

```diff
@@ -7,9 +7,9 @@
 
 import matplotlib.pyplot as plt
 plt.rcParams.update({
+    # Use LaTeX default serif font-family.
+    # https://matplotlib.org/stable/tutorials/text/usetex.html#sphx-glr-tutorials-text-usetex-py
     "font.family": "serif",
-    # Use LaTeX default serif font.
-    "font.serif": [],
```

Comment:
this should not be changed anymore?

Reply:
@FranzForstmayr can you do this small revert? Basically an empty list is valid (and now fixed) for PGF backend.